ansible/roles/mn-createprotx/tasks/main.yml

```diff
@@ -40,6 +40,10 @@
     num_blocks: 1
     balance_needed: 0
 
+- name: Pause for 5 minutes to allow propogation of protx's
+  pause:
+    minutes: 5
```

Comment:
Should we add a condition and pause it only for "larger devnets if minimumblocks is above 4032"?

Reply:
We can, but I'm hoping to get rid of this in the next few weeks and replace it with proper code to check that all protx's have gone through before it starts the next part.

This is just a quick bandaid which me and strophy can comment in and out depending on what's needed over the next few weeks